AoE2.netDesktop/LibAoE2Net/Functions/MatchExt.cs

```diff
@@ -17,7 +17,7 @@ public static class MatchExt
     /// <returns>local time value as DateTime type.</returns>
     public static DateTime GetOpenedTime(this Match match)
     {
-        var ret = DateTimeOffset.FromUnixTimeSeconds(match.Opened ?? 0).LocalDateTime;
+        var ret = DateTimeOffset.FromUnixTimeSeconds(match.Opened ?? 0).UtcDateTime;

```

Comment:
Almost user wants to check it as their local time, so it should be converted to local time.